test/models/mountain_test.rb

```diff
@@ -35,6 +35,18 @@ def setup
     assert @mountain.lists.count >= 2
   end
 
+  test 'should not save a duplicate mountain' do
+    duplicate_mountain = Mountain.new(
```

Comment:
it doesn't save if the duplicated mountain is exactly the same
what happens if only one or a few of the variables are the same? will it still save?